### PR TITLE
Add stress test for bundle in memory

### LINKED ISSUE #3013: Stress test having a large bundle in memory
Add a stress test that has a large bundle as a dependency. This is to catch regressions such as https://github.com/codalab/codalab-worksheets/issues/3011.

### PR DESCRIPTION
**Pull request by teetone**
### Reasons for making this change

We need this stress test to test out https://github.com/codalab/codalab-worksheets/pull/3012

Resolves #3013 

### Checklist

* [x] I've added a screenshot of the changes, if this is a frontend change
* [x] I've added and/or updated tests, if this is a backend change
* [x] I've run the [pre-commit.sh](https://github.com/codalab/codalab-worksheets/blob/master/pre-commit.sh) script
* [x] I've updated [docs](https://github.com/codalab/codalab-worksheets/tree/master/docs), if needed

### CODE REVIEW

**Review by nelson-liu — APPROVED** [review]

File: tests/stress/stress_test.py [hunk 1]
```diff
@@ -161,8 +161,16 @@ def _heartbeat(self):
     def _test_large_bundle(self):
         self._set_worksheet('large_bundles')
         large_file = TestFile('large_file', self._args.large_file_size_gb * 1000)
-        self._run_bundle([self._cl, 'upload', large_file.name()])
+        dependency_uuid = self._run_bundle([self._cl, 'upload', large_file.name()])
```
Comment (percyliang) [inline tests/stress/stress_test.py:166]: Can you document that this needs to be larger than the max memory on the system to test that things are being streamed?

Comment (percyliang) [reply tests/stress/stress_test.py:166]: Also, one way to make tests run faster is to impose artificial memory limits on the process.

File: tests/stress/stress_test.py [hunk 2]
```diff
@@ -303,7 +311,7 @@ def main():
 
     if args.heavy:
         print('Setting the heavy configuration...')
-        args.large_file_size_gb = 10
+        args.large_file_size_gb = 32
```
Comment (nelson-liu) [inline tests/stress/stress_test.py:316]: could set this to `max(10, gigabytes of ram on the machine)`

Comment (teetone) [reply tests/stress/stress_test.py:316]: I'm not sure if it's easy to get the memory of the machine programmatically since this script runs remotely.

**Pull request merged**